packages/Meshfree/test/tstDetailsCommunicationHelpers.cpp

```diff
@@ -155,6 +168,63 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( DetailsDistributedSearchTreeImpl,
                                                 success, out );
 }
 
+TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( DetailsNearestNeighborOperatorImpl, fetch,
+                                   DeviceType )
+{
+    using ExecutionSpace = typename DeviceType::execution_space;
+
+    Teuchos::RCP<const Teuchos::Comm<int>> comm =
+        Teuchos::DefaultComm<int>::getComm();
+    int const comm_rank = comm->getRank();
+    int const comm_size = comm->getSize();
+
+    // make communicaton plan
+    Kokkos::View<int *, DeviceType> indices( "indices", comm_size );
+    Kokkos::View<int *, DeviceType> ranks( "ranks", comm_size );
+    Kokkos::parallel_for( "fill_indices_and_ranks",
```

Comment:
Maybe use `DTK_MARK_REGION` you didn't do in the test. It's up to you, I don't think it's important in the test.

Reply:
I was only providing a label because I thought I had to.  I will either remove or leave as is.